client/components/jetpack/card/jetpack-product-card/index.tsx

```diff
@@ -249,7 +268,7 @@ const JetpackProductCard: React.FC< Props > = ( {
 						primary={ buttonPrimary }
 						className="jetpack-product-card__button"
 						onClick={ onButtonClick }
-						disabled={ isDisabled }
+						disabled={ isDisabled || isDeprecated }
```

Comment:
Seems like `isDisabled` can never be true due to the fact that this is wrapped in `! isDisabled && ()` condition.

Reply:
Yes, this is confusing. If `isDisabled` is true, we won't render the button. However, if `isDisabled` is `false` and `isDeprecated` is `true`, we will render a disabled button.

We might have to add another prop called `isButtonVisible` (or something like that) and use that prop to render/hide the button.

In any case, I'll add a card on our maintenance board to fix this.

Thanks for pointing out this issue.

Reply:
Yeah, just to clarify, at a first glance, this looks as simple as just altering it to just be `isDeprecated`, because of this check: https://github.com/Automattic/wp-calypso/pull/52341/files#diff-4497eb96889885ef2d803f05ccfe30d8c8fcb97313ff9cf3b19e8b9d3f0f0e47R266